usaspending_api/api_contracts/contracts/award/idv/FundingMetaData.md

```diff
@@ -0,0 +1,22 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# IDV Funding Roll-up [/api/v2/awards/idvs/funding_rollup/]
+
+This end point returns award metadata specifying the total transaction obligations, awarding agencies, and federal accounts.
+
+## POST
+
++ Request (application/json)
+    + Attributes (object)
+        + award_id: `CONT_AW_9700_-NONE-_N0018918D0057_-NONE-` (required, string)
++ Response 200 (application/json)
+    + Attributes
+        + results (required, array[IDVFundingRollUpResponse])
```

Comment:
As we discussed in Slack, this doesn't need to be an array since there will only be one result.